<launch>
  <arg name="camera_ns" default="aravis_cam"/>

  <node type="detection_to_pose" pkg="rocket_league_estimation" name="map_detection_to_pose">
    <param name="tag_id"    value="69"/>
    <param name="tag_name"  value="map_tag"/>
    <param name="body_name" value="map"/>

    <param name="publish_tf" value="true"/>
    <param name="publish_pose" value="false"/>

    <remap from="tag_detections" to="$(arg camera_ns)/tag_detections"/>
  </node>
  
  <node pkg="tf2_ros" type="static_transform_publisher" name="map_tag_publisher" args="0 0 0 0 0 0 map map_tag" />
</launch>
